samples/libcurl/compile_windows.ps1

```diff
@@ -9,7 +9,7 @@ jextract `
   --dump-includes 'includes_all.conf' `
   "$curlpath\include\curl\curl.h"
   
-Select-String -Path 'includes_all.conf' -Pattern 'curl' | %{ $_.Line } | Out-File -FilePath 'includes_filtered.conf' -Encoding ascii
+Select-String -Path 'includes_all.conf' -Pattern '(curl|sockaddr )' | %{ $_.Line } | Out-File -FilePath 'includes_filtered.conf' -Encoding ascii

```

Comment:
This was an existing issue that went unnoticed previously. The curl_sockaddr type depends on the Windows sockaddr type now (due to the layout referencing it directly), so we need to include that as well.